src/Pages/pokemonDetail.jsx

```diff
@@ -0,0 +1,48 @@
+import { useParams } from "react-router-dom";
+import { gql, useQuery } from "@apollo/client";
+import { Link } from "react-router-dom";
+
+const pokemonDetail = gql`
+  query samplePokeAPIquery($PokemonID: Int!) {
+    gen3_species: pokemon_v2_pokemonspecies(
+      where: { id: { _eq: $PokemonID } }
+    ) {
+      name
+      id
+      is_mythical
+      is_baby
+      pokemon_v2_pokemonspeciesflavortexts(
+        limit: 1
+        order_by: { version_id: desc }
+        where: { language_id: { _eq: 9 } }
+      ) {
+        flavor_text
+        language_id
+        version_id
+      }
+    }
+  }
+`;
+
+function PokemonDetail() {
+  let { id } = useParams();
```

Comment:
Nit: `const` can be used instead of `let` here since `id` is not reassigned at any point.

```suggestion
  const { id } = useParams();
```